WalletWasabi/Models/ConcurrentHashSet.cs

```diff
@@ -16,7 +16,7 @@ namespace WalletWasabi.Models
 	/// concurrently from multiple threads.
 	/// https://github.com/i3arnon/ConcurrentHashSet/
 	/// </remarks>
-	[DebuggerDisplay("Count = {Count}")]
+	[DebuggerDisplay("Count = {" + nameof(Count) + "}")]
```

Comment:
I don't like these ones. ACK for the rest.

Reply:
I wanted to use string interpolation but then I had to escape the curly braces :/. I'll remove these ones.

Reply:
Yeah, I spent like 20 minutes trying to do that until I gave up :D